tst/com/amazon/corretto/crypto/provider/test/AesTest.java

```diff
@@ -579,6 +581,24 @@ public void test_initParameters() throws Throwable {
         assertArrayEquals(PLAINTEXT, decrypted);
     }
 
+    @SuppressWarnings("ConstantConditions")
```

Comment:
I'm not familiar with this one. Not an objection, but what is it?

Reply:
Ah, this just tells IntelliJ IDEA not to complain that `key` is `null` everywhere in the method body.